application/classes/Ushahidi/Console/Notification/Collection.php

```diff
@@ -45,8 +52,8 @@ public function setNotificationQueueRepo(NotificationQueueRepository $repo)
 	protected function configure()
 	{
 		$this
-			->setName('notification')
-			->setDescription('Manage notifications')
+			->setName('notification:collection')
+			->setDescription('Manage notifications for Collections')
 			->addArgument('action', InputArgument::OPTIONAL, 'list, queue', 'list')
```

Comment:
I'm leaning towards removing the `action` argument from all our tasks. You can just override `execute` entirely instead..
I've done that for `config:set` .. what do you think?